packages/trader/src/Modules/Trading/Components/Form/TradeParams/strike.tsx

```diff
@@ -121,7 +122,7 @@ const Strike = observer(() => {
             </DesktopWrapper>
             <MobileWrapper>
                 <div className='mobile-widget__wrapper'>
-                    <div className='strike-widget' onClick={toggleWidget}>
+                    <div className='strike-widget' onClick={toggleWidget} onKeyDown={toggleWidget}>
```

Comment:
Can we add a key check? Maybe check if the pressed key is `Enter` before calling toggleWidget.

Reply:
No need, this is a fake event, it's added only on mobile and you don't have a keyboard on mobile

Reply:
In this PR https://github.com/binary-com/deriv-app/pull/11218 I created some kind of wrapper for such functions. In is located in `@shared package` and called `clickAndKeyEventHandler`. You can pass your `toggleWidget` as a callback and my wrapper will check pressed key.

Reply:
Maybe it should be a button? If it contains Icon 'cross' and is clickable, it might be better for semantic + will be no need in adding onKeyDown for mobile just to satisfy sonarCloud

Reply:
Yes, it shouldn't be a button. I will add the wrapper but I mean it's a no use anyways, I would rather take down this whole sonarcloud thing whatsoeve. It's supposed to help improve our accessibility. First of all, there's no such thing as accessibility in the first place in deriv-app, and second it makes us add a keyboard events for mobile users, which is same as giving an empty kettle to a thirsty person in a desert.